src/api/seasons.go

```diff
@@ -187,9 +191,12 @@ func updateGameDispute(w http.ResponseWriter, r *http.Request, seasonId string,
 	"%s -- Automated Dispute Notification"
 	`
 	
+	var emailList []string
+	emailList = []string{"dungeongod@gmail.com"}
```

Comment:
Would prefer to have this be a concatenated string to break up the email, so it is harder for spam bots to pick it up when trolling web pages.